cmd/check.go

```diff
@@ -24,7 +24,6 @@ func setupCheckCommand() *cobraext.Command {
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			err := cobraext.ComposeCommands(args,
-				setupFormatCommand(),
```

Comment:
We can make this change in a different PR: https://github.com/elastic/elastic-package/pull/1936